### PR TITLE
Add new Corporate Action samples

### PR DESCRIPTION
**Pull request by ayxue**
Hi Bas,

Here are new samples of Corporate Actions from Ray.  Please review. 

Leave any comments and I will change codes accordingly.

/Ray Xue

### CODE REVIEW

**Review by basgroot — CHANGES_REQUESTED** [review]

File: corporate-actions/elections/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Client-side Samples for placing Stock Orders
```
[the remark this answers is not in the document]
Comment (basgroot) [reply corporate-actions/elections/README.md:1]: Can you summarize the difference between elections, events, holdings and standing instructions?

File: corporate-actions/elections/demo.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    }
+
+    /**
+     * This is an example of getting instrument details, option or future series.
+     * @return {void}
+     */
+    function elect() {
+        const electBody = document.getElementById("idElectBody").value;
+        let urlPath = "/ca/v2/elections";
+        var body = JSON.parse(electBody);
```
[the remark this answers is not in the document]
Comment (ayxue) [reply corporate-actions/elections/demo.js:69]: I have incorrectly commented this function so that it is not for getting an election object but do an action called elect. So I have changed the comment but keep the function name as elect

**Review by basgroot — APPROVED** [review]
